unic/ucd/case/src/changes_when_uppercased.rs

```diff
@@ -0,0 +1,106 @@
+// Copyright 2015 The Servo Project Developers.
```

Comment:
Just a small question: what part of this file comes from Servo libraries? I'm supposing the tests?